src/main/java/org/shanerx/tradeshop/object/BukkitVersion.java

```diff
@@ -0,0 +1,82 @@
+/*
+ *     Copyright (c) 2016-2017 SparklingComet @ http://shanerx.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * NOTICE: All modifications made by others to the source code belong
+ * to the respective contributor. No contributor should be held liable for
+ * any damages of any kind, whether be material or moral, which were
+ * caused by their contribution(s) to the project. See the full License for more information
+ */
+
+package org.shanerx.tradeshop.object;
+
+import org.bukkit.Bukkit;
+
+public class BukkitVersion {
+    private final String SERVERVERSION = Bukkit.getBukkitVersion();
+
+    private double major;
+    private int minor;
+
+    public BukkitVersion() {
+        String[] ver = SERVERVERSION.split("-")[0].split("\\.");
+        this.major = parseDouble(ver[0]) + parseMinor(ver[1]);
+        this.minor = parseInt(ver[2]);
+    }
+
+    public BukkitVersion(double major, int minor) {
+        this.major = major;
+        this.minor = minor;
+    }
+
+    public String toString() {
+        return major + "." + minor;
+    }
+
+    public String getFullVersion() {
+        return SERVERVERSION;
+    }
+
+    public double getMajor() {
+        return major;
+    }
+
+    public int getMinor() {
+        return minor;
+    }
+
+    public Boolean isBelow19() {
```

Comment:
how about we implement a method with the following signature:
`public boolean isAbove(double version);` ?

Reply:
Also why are we using the boxed primitives? (`Boolean` instead of `boolean`)